modules/python/init.zsh

```diff
@@ -46,6 +46,20 @@ if (( $? && $+commands[virtualenvwrapper.sh] )); then
   source "$commands[virtualenvwrapper.sh]"
 fi
 
+# Load PIP completion.
+if (( $+commands[pip] )); then
+  cache_file="${0:h}/cache.zsh"
+
+  if [[ "$commands[pip]" -nt "$cache_file" || ! -s "$cache_file" ]]; then
+    # pip is slow; cache its output. And also support 'pip2', 'pip3' variants
+    pip completion --zsh | sed -e "s|compctl -K _pip_completion pip|& pip2 pip3|" >! "$cache_file" 2> /dev/null
```

Comment:
1. `pip completion --zsh` just generated the stub function `function _pip_completion {}`. And it is identical regardless of `pip` version. (Try comparing `pip2 completion --zsh` and `pip3 completion --zsh`).

2. While both `pip2` and `pip3` might be available/installed, almost certainly one of them would be soft-linked to `pip`. And that's all that we need to generate and source in.

3. All that we are doing here is:
    - Sourcing in `_pip_completion pip()`
    - Tweaking `compctl -K _pip_completion pip` to work for `pip2` and `pip3` as well.